src/main/java/io/cdap/delta/datastream/util/Utils.java

```diff
@@ -19,75 +19,35 @@
 
 import com.google.api.gax.core.CredentialsProvider;
 import com.google.api.gax.longrunning.OperationFuture;
-import com.google.api.gax.rpc.AbortedException;
-import com.google.api.gax.rpc.AlreadyExistsException;
-import com.google.api.gax.rpc.FailedPreconditionException;
-import com.google.api.gax.rpc.InvalidArgumentException;
-import com.google.api.gax.rpc.NotFoundException;
+import com.google.api.gax.rpc.*;
 import com.google.auth.Credentials;
 import com.google.auth.oauth2.GoogleCredentials;
-import com.google.cloud.datastream.v1alpha1.AvroFileFormat;
```

Comment:
Auto optimization of imports by IDE, have configured the cdap coding standard https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard